index.html

```diff
@@ -17,7 +17,7 @@
            <ul class="nav-items">
```

Comment:
- [ ] Please add the link to the live-demo of your portfolio to your readme.md file.

- [Optional] Please consider making your desktop version match the template. I think it is centered because you used a fixed width on the wrapper. Try using `.works {
    width: 90%;
    margin: auto;
    background: transparent;
}`, `.wrapper { width: 100%; }` in the media query for the desktop version.